Sources/Brave/Frontend/Browser/BrowserViewController/BrowserViewController+ToolbarDelegate.swift

```diff
@@ -108,7 +109,29 @@ extension BrowserViewController: TopToolbarDelegate {
   }
 
   func topToolbarDidPressReload(_ topToolbar: TopToolbarView) {
-    tabManager.selectedTab?.reload()
+    if WalletFeatureFlags.SNSDomainResolverEnabled, let url = topToolbar.currentURL, url.schemelessAbsoluteDisplayString.endsWithSupportedSNSExtension, let currentStatus = Preferences.Wallet.Web3DomainOption(rawValue: Preferences.Wallet.resolveSNSDomainNames.value) {
```

Comment:
`schemelessAbsoluteDisplayString` might not work?
If the url is `https://onybase.sol/some_path/some_other_path/something?meh=foo` then it will NOT `endWithSupportedSNSExtension` since that checks for `prefix` with the query URL and path and all.
It should be checking if the `eTLD+1` or host/domain contains that prefix.

Reply:
added `domainURL` in [7f2c61e](https://github.com/brave/brave-ios/pull/6906/commits/7f2c61e1159ed7c50d4af04bffb6c19a9c7bc262)